spec/lib/lighthouse/facilities/client_spec.rb

```diff
@@ -0,0 +1,151 @@
+  let(:params) do
+    {
+      bbox: [60.99, 10.54, 180.00, 20.55] # includes the Phillipines and Guam
+    }
+  end
+
+  let(:vha_358_attributes) do
+    {
+      id: 'vha_358',
+      type: 'va_facilities',
+      name: 'Manila VA Clinic',
+      facility_type: 'va_health_facility',
+      classification: 'Other Outpatient Services (OOS)',
+      website: nil,
+      lat: 14.544080000000065,
+      long: 120.99139000000002,
+      address: {
+        'mailing' => {},
+        'physical' => {
+          'address_1' => '1501 Roxas Boulevard',
+          'address_2' => 'NOX3 Seafront Compound',
+          'address_3' => nil,
+          'city' => 'Pasay City',
+          'state' => 'PH',
+          'zip' => '01302'
+        }
+      },
+      phone: {
+        'after_hours' => '000-000-0000',
+        'enrollment_coordinator' => '632-550-3888 x3780',
+        'fax' => '632-310-5962',
+        'main' => '632-550-3888',
+        'patient_advocate' => '632-550-3888 x3716',
+        'pharmacy' => '632-550-3888 x5029'
+      },
+      hours: {
+        'friday' => '730AM-430PM',
+        'monday' => '730AM-430PM',
+        'saturday' => 'Closed',
+        'sunday' => 'Closed',
+        'thursday' => '730AM-430PM',
+        'tuesday' => '730AM-430PM',
+        'wednesday' => '730AM-430PM'
+      },
+      services: { 'health' => %w[Audiology Cardiology Dermatology EmergencyCare Gastroenterology
+                                 MentalHealthCare Ophthalmology Orthopedics PrimaryCare SpecialtyCare],
+                  'last_updated' => '2020-04-06', 'other' => [] },
+      feedback: {
+        'effective_date' => '2019-06-20',
+        'health' => { 'specialty_care_routine' => 0.9100000262260437 }
+      },
+      access: {
+        'effective_date' => '2020-04-06',
+        'health' => [
+          { 'established' => 14.352941, 'new' => 158.0, 'service' => 'Audiology' },
+          { 'established' => 34.034482, 'new' => 66.75, 'service' => 'Cardiology' },
+          { 'established' => 3.4, 'new' => 123.5, 'service' => 'Dermatology' },
+          { 'established' => nil, 'new' => 208.0, 'service' => 'Gastroenterology' },
+          { 'established' => 24.228571, 'new' => 134.222222, 'service' => 'MentalHealthCare' },
+          { 'established' => 10.111111, 'new' => 154.6, 'service' => 'Ophthalmology' },
+          { 'established' => 25.17647, 'new' => 122.0, 'service' => 'Orthopedics' },
+          { 'established' => 18.927083, 'new' => 28.8125, 'service' => 'PrimaryCare' },
+          { 'established' => 19.22807, 'new' => 75.317073, 'service' => 'SpecialtyCare' }
+        ]
+      },
+      mobile: false,
+      active_status: 'A',
+      visn: '21',
+      operating_status: { 'code' => 'NORMAL' },
+      facility_type_prefix: 'vha',
+      unique_id: '358'
+    }
+  end
+
+  it 'is an Facilities::Client object' do
+    facilities_client = described_class.new('fake_key')
+    expect(facilities_client).to be_an(Lighthouse::Facilities::Client)
```

Comment:
What are we testing here? how rspec describes a class?

Reply:
Just a sanity check that the client is being created. I stole it from the PPMS client spec.

Reply:
Ya, I'd rather not cargo cult from that.